### DISCUSSION TITLE
Add verified checkmark for GitHub users

### DISCUSSION THREAD

**Discussion by ArjunSharda** [community/community#27643] [Other Feature Feedback, Questions, & Ideas]
GitHub users are currently not able to be verified, and there are a ton of people who can impersonate other celebrity users. There should be an option for requesting verification just like for GitHub organizations but that the message would be "This person is a verified public figure.".

**Comment by rowan-mcalpin** [comment]
Or even just verifying you are who you say you are, even if you're not a public figure. Just send a photo of your ID.

**Reply by ArjunSharda** [reply]
Maybe there could be a different badge for that, like a gray-type verified.

**Reply by rowan-mcalpin** [reply]
Take instagram, anyone can request to be verified even if they aren't celebrities, and it shows up the same (but doesn't happen nearly as often)

**Reply by ArjunSharda** [reply]
Following up, this actually would be a great solution. I think that would work. Sorry for late reply.

**Reply by rowan-mcalpin** [reply]
A year later, to clarify the intent with this, it would be to prove you are who you say you are. Even if you're a nobody that nobody knows about, as long as you prove that you are *you*, then your account should be verified. Maybe revoke verification if the username is changed?

**Comment by ArjunSharda** [comment]
That’s a good idea but it can be limited to public figures / known people /
celebrities.

**Comment by jfullstackdev** [comment]
I'm not against this, but what would be the benefit of verifying a user ? if you know in the first place, it's suspicious ? 

take Linus Torvalds, the creator of Linux OS and Git, 

anyone can copy him and his works because it's open source, yet anybody knows him even without verifying himself

<https://github.com/torvalds>

![image](https://user-images.githubusercontent.com/47092464/181875408-70501c92-b26d-459a-8f95-a9824a699c0d.png)

take another example in the Crypto world, we all know in the Crypto world that celebrities are being used to believe others theirs are legit while in fact it's all lie, others would even create a very reliable website when they will be simply gone when things go wrong

**Comment by ArjunSharda** [comment]
Imagine people with the same time profile picture pretending to be someone
else. This could help for in the search.

**Reply by jfullstackdev** [reply]
but as you can see in the above example, it's not happening, users can tell who is the legit Linus Torvalds, even if there are fake accounts

**Reply by ArjunSharda** [reply]
Ah alright then verification can be for everyone to verify they are not a bot.

**Reply by jfullstackdev** [reply]
check this out

https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#3-account-requirements

GitHub does not allow bots to create other bots account, so it's the responsibility of the user who created that machine account

again I'm not against this verification request, just want to discuss things

**Reply by ArjunSharda** [reply]
I completely get what you mean for bot accounts and stuff but people just can create accounts and use it for malicious purposes. Verifying people can help people know that someone has actually been verified as a real person and it makes it more trustable. This could even for the GitHub marketplace be a good addition to show / OAUTH apps.

**Comment by KodeyThomas** [comment]
GPG Keys achieve this, they verify the person who is commiting using that email address is the real person. Take one of my commits as an example.
![Screenshot 2022-09-29 at 18 29 33](https://user-images.githubusercontent.com/13765713/193101723-2b140eb4-65bd-443a-8c65-8060e5dfdc90.png)
